theme/lumo/vaadin-menu-bar-item-styles.html

```diff
@@ -0,0 +1,24 @@
+<link rel="import" href="../../../vaadin-lumo-styles/sizing.html">
+
+<dom-module id="lumo-menu-bar-item" theme-for="vaadin-context-menu-item">
+  <template>
+    <style>
+      :host([theme="menu-bar-item"]) [part="content"] {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      :host([theme="menu-bar-item"]) [part="content"] ::slotted(iron-icon) {
+        display: inline-block;
+        width: var(--lumo-icon-size-m);
+        height: var(--lumo-icon-size-m);
+      }
+
+      :host([theme="menu-bar-item"]) [part="content"] ::slotted(iron-icon[icon^="vaadin:"]) {
+        padding: 0.25em;
```

Comment:
Probably use lumo-variable? `--lumo-space-xs` / `--lumo-space-s`. Not a blocker.

Reply:
Done.